frameworks/digital-outcomes-and-specialists/questions/briefs/niceToHaveRequirementsParticipants.yml

```diff
@@ -1,23 +1,11 @@
-name: 'Technical competence: nice-to-have skills and experience'
+name: 'Nice-to-have skills and experience'
 id: niceToHaveRequirements
-question: 'Technical competence: nice-to-have skills and experience'
+question: 'Nice-to-have skills and experience'
 question_advice: |
-  To find the best supplier to meet your needs, you should:
+  If too many suppliers have all the essential skills and experience, use the nice-to-have skills and experience to exclude suppliers.
 
-  - list as many types of skills and experience as you can
-  - be specific about the skills and experience you're looking for
-
-  If too many suppliers have all the essential skills and experience, you’ll use the nice-to-have skills and
-  experience to exclude suppliers.
-
-  You’ll need to use the skills and experience you list here to evaluate suppliers’ technical competence.
-
-  You must tell suppliers if you want to score individual criteria.
-
-  [Read more about how to set evaluation criteria](https://www.gov.uk/guidance/how-to-set-your-evaluation-criteria-when-buying-digital-outcomes-and-specialists-services#giving-points-to-individual-criteria)
-
-  List the skills and experience you’d like the supplier to have,
-  eg have experience recruiting users for government research
+  List the skills and experience you’d like the supplier to have,<br />
+  eg have experience recruiting participants with hernias.
```

Comment:
Would `experience recruiting participants from the armed forces` be a bit less weird?  I don't know...